play/PlayerRules/Basic Rules.txt

```diff
@@ -581,13 +604,14 @@ to roll either for (15 - FORT) turns, or until they pass.
 Conditions:
 Bleeding - d10/2+1 (drop the remainder) stackable damage per turn. Broken with 
 Medicine Skill check. DC varies. If you have a bleed condition, seek medical aid
-immediately, You CAN Bleed Out, you have been warned.
+immediately, You can Bleed Out.

```

Comment:
I understand, I like jokes in the rules, too. Jokes can be difficult though, the reader has to interpret the meaning without any cues.
I didn't take this as a joke at all, it came across as unnecessary emphasis.